rust/garbage_collector/src/operators/list_files_at_version.rs

```diff
@@ -149,13 +145,10 @@ impl Operator<ListFilesAtVersionInput, ListFilesAtVersionOutput> for ListFilesAt
                     }
                     Err(e) => Err(e),
                 }
-            });
-        }
+            }).buffer_unordered(32);
```

Comment:
review: is there a more principled way to think about the right limit? or should we just rely on the NAC to limit appropriately?

Reply:
that's what I use as a guiding principle currently - relying on NAC

Reply:
so do you think I should remove the 32 constant for now? I don't have proof that this particular area is causing any issues

Reply:
decided to remove the constant for now